server/src/app.js

```diff
@@ -5,20 +5,31 @@ const paramPluck = require("./middleware/paramPluck");
 const {
   getOneProject,
   getUserImage,
+  createOneProject
 } = require("./controllers/projectController");
 
-const { getRewards } = require("./controllers/rewardController");
+const { getRewards, createOneReward, updateOneReward, deleteOneReward  } = require("./controllers/rewardController");
```

Comment:
Good modularity by putting the functions in a separate file

Reply:
Again this is Zach breaking out the Query functions into a separate file. I am a big fan of this and would have probably done this anyway.